### ISSUE TITLE
Send an email the first time a sync isn't working anymore

### ISSUE THREAD

**Issue by bnjbvr** [bnjbvr/kresus#233]
(because the password is still encrypted, e.g.)

**Comment by ZeHiro** [comment]
Can this be closed with 0bb50b2d14a258607e54aebed823734ee1941456 ? 
Or should we also send an email when NO_PASSWORD is set?

**Comment by bnjbvr** [comment]
Yeah, I was merely thinking about sending an email when there's no password set. Also, when the user sets the password by hand, they shouldn't have to do the manual fetch at all, in my opinion.

**Comment by ZeHiro** [comment]
I agree for the auto fetch

**Comment by ZeHiro** [comment]
- [x] Send email to q user when no password is set ( #420)
- [x]  Fetch accounts and operations on access password update ( #420 )
- [ ] Client counter part (getting fetches accounts and operations)
